main.tf

```diff
@@ -45,12 +45,10 @@ resource "google_project_iam_binding" "spot-account-iam" {
 }
 
 # Authoritative for a given role. Updates the IAM policy to grant a role to a list of members. Other roles within the IAM policy for the project are preserved.
```

Comment:
I guess, this message is no longer valid.

Reply:
Updated message.